azerothcore/azerothcore-wotlk
# fix(Scripts/Temple of AhnQiraj): Qiraji Champion.

Fixes #12933



**Test it with #12862**

## Issues Addressed:

- Closes #12933

## Tests Performed:

- Not tested.

## How to Test the Changes:

`.go c id 15252`
check if spells are cast
kill their formation members.
see if it casts Vengeance once all are dead.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
